packages/web-components/src/components/ic-top-navigation/ic-top-navigation.tsx

```diff
@@ -77,6 +77,11 @@ export class TopNavigation {
    */
   @Prop() inline: boolean = false;
 
+  /**
+   * The short title of the app to be displayed at small screen sizes in place of the regular app title.
+   */
+  @Prop() shortTitle: string = "";
```

Comment:
Should this be renamed to `shortAppTitle` because that is what it will be replacing?